hack/build-s2i-dpdk.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+ELAPSED=0
+TIMEOUT=180
+ATTEMPTS=0
+MAX_ATTEMPTS=10
```

Comment:
you can use a single ATTEMPTS var if you start from 10 and decrease on each try until zero.
I think it's more readable.
ie in the until do
[ $ATTEPTS -eq "0" ]
and later below do
{{ ATTEMPTS-- }}

actually same can go for timeout